packages/access-api/src/config.js

```diff
@@ -29,11 +32,14 @@ export function loadConfig(env) {
     }
   }
 
+  const DID = env.DID
+  const PRIVATE_KEY = vars.PRIVATE_KEY
+  const signer = configureSigner({ DID, PRIVATE_KEY })
   return {
     DEBUG: boolValue(vars.DEBUG),
     ENV: parseRuntimeEnv(vars.ENV),
 
-    PRIVATE_KEY: vars.PRIVATE_KEY,
+    PRIVATE_KEY,
```

Comment:
we probably dont need this one anymore right ?

Reply:
https://github.com/web3-storage/w3protocol/pull/275/commits/852951f932d97fc651c08258fdc6f4a7b15ce658